packages/Button/stories/Button.mdx

```diff
@@ -0,0 +1,234 @@
+import { Meta, Props, Story } from "@storybook/addon-docs/blocks";
+import PlusIcon from "@paprika/icon/lib/Add";
+import InfoIcon from "@paprika/icon/lib/InfoCircle";
+import ButtonSubmit from "./examples/ButtonSubmit";
+import CloseButtonRef from "./examples/CloseButtonRef";
+import Focus from "./examples/Focus";
+import NewRef from "./examples/NewRef";
+import OldRef from "./examples/OldRef";
+import Button from "../src/";
+
+## More Examples
+
+### Index
+
+- <a href="#basic-button">Basic Button</a>
+
+  - <a href="#kinds">Kinds</a>
+
+  - <a href="#sizes">Sizes</a>
+
+  - <a href="#states">States</a>
+
+  - <a href="#submit">Submit</a>
+
+- <a href="#close-button">Close Button</a>
+- <a href="#link-button">Link Button</a>
+- <a href="#icon-button">Icon Button</a>
+
+### Basic Button
+
+To add a basic button import the button and add the `<Button>` component to your project.
+
+```js
+<Button>Click Me!</Button>
+```
+
+##### Result
+
+<Button>Click Me!</Button>
```

Comment:
I think this is redundant – we have a real basic Usage example above in the README. Lets go straight into the Kinds example.